.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,21 @@
+# This workflow will publish a package to npmjs.com when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Node.js Package
+
+on: push
+
+jobs:
+  publish-npm:
+    if: github.ref == 'refs/heads/master' && github.event_name == 'push' && contains(github.event.head_commit.message, 'chore(release)')
```

Comment:
I guess we could use:
```
on:
  push:
    branches:
      - master
 ```
instead of checking for the `master` branch and `push` event in the `if`. It seems slightly more concise to me. Is there any advantage of the current solution over that?

Reply:
I never want to release each change in the master branch automatically. You should have control over the whole release cycle and what's included in each release. That is why it is stricter.

We can change that, that's how we approach it when reviewing what will be released in each new SDK release.

Reply:
I didn't mean removing the whole `if` altogether, but rather doing something like that:
```yaml
on:
  push:
    branches:
      - master
 
jobs:
  publish-npm:
    if: contains(github.event.head_commit.message, 'chore(release)') 
```

Which is leaving the same logic, but implementing it slightly different. The `if: github.ref == 'refs/heads/master' && github.event_name == 'push'` seemed to me like reinventing a wheel that is provided by the:
```yaml
on:
  push:
    branches:
      - master
 ```

Sorry if that wasn't clear in my original comment.

Reply:
Oh, I understand. I'm not sure whether the two conditions apply together or are mutually exclusive in this case. I'd have to check.

Reply:
@prokopsimek Were you able to check that?

Reply:
@Toreno96 Hi Daniel, I'll check it soon. We also realized that there might be a better way for this SDK release, will post update soon!